ui/src/layouts/storage.rs

```diff
@@ -30,9 +30,13 @@ use crate::{
     STATIC_ARGS, WARP_CMD_CH,
 };
 
+const ROOT_DIR_NAME: &str = "root";
```

Comment:
Can we make this a public static? I like the idea of making this a const because it will be easier to change it in the future versions if need be. This could also make it useful to other developers to use outside of the file.